src/CMakeLists.txt

```diff
@@ -303,6 +303,19 @@ if(${ENABLE_COVERAGE})
   list(APPEND EXTRALIBS gcov)
 endif(${ENABLE_COVERAGE})
 
+# libstdc++ uses TBB to implement <execution> if it is available,
+# which means that if we're going to use <execution>, we need to link
+# against TBB if it's available.
+#
+# It happens to work by accident at present because the optimizer
+# optimizes out our one reference, for now, but this breaks compiling
+# with `-O0` and will break anyway once we try to do more.
+find_package(TBB QUIET)
+if(${TBB_FOUND})
+  message(STATUS "Linking to TBB for implementations of <execution>.")
+  list(APPEND EXTRALIBS TBB::tbb)
```

Comment:
also, can we link `object_io_exerciser` to `TBB::tbb` instead? as `io_exerciser/ObjectModel.cc` is the only source file including `<execution>`. more explicit this way.